weave/trace_server/calls_query_builder.py

```diff
@@ -427,11 +430,36 @@ def as_sql(self, pb: ParamBuilder, table_alias: str = "calls_merged") -> str:
             outer_query.limit = self.limit
             outer_query.offset = self.offset
 
+        # because we are selecting from a subquery, we need to prefix the fields, but we dont want to run all aggregations again
+        # inparticular argMaxMerge throws an error when run twice on display_name
+        # We also filter out summary_dump, because we add costs to summary dump in the select statement
+        final_select_fields = [
```

Comment:
it seems like most of this should happen inside of `cost_query` ... since you already are passing down the `all_calls`. That way the exclusion of summary_dump is close to where it gets excluded